src/sidebar/annotation-mapper.js

```diff
@@ -12,7 +12,7 @@ function getExistingAnnotation(annotationUI, id) {
 
 // Wraps the annotation store to trigger events for the CRUD actions
 // @ngInject
-function annotationMapper($rootScope, annotationUI, store) {
+function annotationMapper($rootScope, annotationUI, api) {
```

Comment:
Wahoo! Comment from the peanut gallery: I LIKE THIS CHANGE! Yay for self-documenting naming.